lib/commanded/event/handler.ex

```diff
@@ -626,12 +626,38 @@ defmodule Commanded.Event.Handler do
       @doc false
       def before_reset, do: :ok
 
-      defoverridable init: 1, after_start: 1, before_reset: 0
+      @doc false
+      def error(error, failed_event, failure_context) do
+        retry_strategy = Keyword.get(unquote(using_opts), :retry, :stop)
+        Handler.handle_error(retry_strategy, error, failed_event, failure_context)
+      end
+
+      defoverridable init: 1, after_start: 1, before_reset: 0, error: 3
+    end
+  end
+
+  def handle_error(:stop, {:error, reason}, _failed_event, _failure_context) do
+    {:stop, reason}
+  end
+
+  def handle_error(:backoff, {:error, reason}, _event, %FailureContext{context: context}) do
```

Comment:
```suggestion
  def handle_error(:random_backoff, {:error, reason}, _event, %FailureContext{context: context}) do
```
Should the name be a bit more precise?

Reply:
I don't think so, as the caller, I would just want to specify that I want it to backoff. I would either expect jitter to be taken care of for me, or I wouldn't even know what that is.

Reply:
Renamed `on_error:`